### DISCUSSION TITLE
Feature Request: Create folders

### DISCUSSION THREAD

**Discussion by dedouwe26** [community/community#112463] [Mobile] [answered]
Hello, I have a feature request: You can create and edit files, but you can’t create or rename folders, so maybe you can add this?
Thanks, 0xDED

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.152.0 (125147271)
Device: iPhone 12 Pro (iOS 17.3.1)
</details>

**Comment by taki-on** [comment]
Thank you for your suggestion @dedouwe26 ! It is a great idea. I will put it in our backlog and discuss it with the team!

**Comment by haldun** [comment, marked as answer]
Hey @dedouwe26 you can create folders by entering `new_folder_name/file_name` to the input when you tap on the Create File menu item.
